src/components/common/StaffPayment.vue

```diff
@@ -1,14 +1,17 @@
 <template>
   <section id="staff-payment">
     <header>
-      <h2>3. Staff Payment</h2>
+      <h2>4. Staff Payment</h2>
```

Comment:
Are there any condition sections above this that might affect this number?

Reply:
Not that i'm aware of. 
Section 3 & 4 currently only render for staff.

Reply:
Perfect.